consensus/consensus.md

```diff
@@ -400,7 +400,7 @@ DesiredRate              = 1 / DesiredTimeBetweenBlocks
 ```
 
 The basic idea of the algorithm is to estimate the current network capacity
-based on the `N` (= 10) previous blocks and use that to set the new
+based on the `N` (= 17) previous blocks and use that to set the new
```

Comment:
This line is implicit on [which block the target adjustment starts at (that seems from block at height 19 - included - the target is adjusted)](https://github.com/aeternity/epoch/pull/1460/files?utf8=%E2%9C%93&diff=split&w=1#diff-4ac652308bacbca71e5f388759d362c1R369). Making it explicit somewhere in the document would remove a little room for ambiguity.

And it seems that idea is excluding blocks 0 (genesis) and 1 from target adjustment?

(This document already details that genesis has height zero - that is good.)